centreon/www/front_src/src/federatedModules/Load/index.tsx

```diff
@@ -130,7 +130,7 @@ export const Remote = ({
   }
 
   return (
-    <MemoizedLoadComponent
+    <LoadComponent
```

Comment:
```suggestion
    <MemoizedLoadComponent
```

Reply:
federated component should update according to all the props i pass them like displayModal , displayButtonConfiguration ..